config.toml

```diff
@@ -48,4 +49,16 @@ publish_hold = [
     "/topics/internals/",
     "/topics/protocol/",
     "/topics/rdd/"
+]
+
+recommended_clients_paths = [
```

Comment:
Two questions here:
- `config.toml` is a site-level configuration (e.g. included on _every_ page). Is there a better place for this data? I would guess either the macro can load a arbitrary file or include a recommended flag in the JSON file itself. It could also be stored in custom frontmatter (see #167), which might be a flexible alternative to present different recommendations on different pages. 
- How was the 'recommendation' list arrived at?

Reply:
- Seems like the custom frontmatter is the best option here as it would also help with [avoiding checking for slug](https://github.com/valkey-io/valkey-io.github.io/pull/166#discussion_r1862587794)
- about the list - https://github.com/valkey-io/valkey-doc/pull/164#discussion_r1862496003

Reply:
Alright @liorsve, we're dependent on #164 make sense.